### PR TITLE
[Docs] Update "Reset" to "Clear"

### PR DESCRIPTION
**Pull request by nataliekwong**
## What
- Updates "Reset" page to say "Clear"
- Updates any open migration guides that reference "Reset". Some additional text formatting changes were made to make the language more explicit and consistent.

### CODE REVIEW

**Comment by nataliekwong** [comment]
just as fyi cc @davinchia @evantahler

File: docs/operator-guides/reset.md [hunk 1]
```diff
@@ -2,29 +2,29 @@
...
 
-Airbyte allows you to reset all streams in the connection, some, or only a single stream (when the connector support per-stream operations).
+Airbyte allows you to clear all streams in the connection, some, or only a single stream.
 
-A sync will automatically start after a completed reset, which commonly backfills all historical data.
+In order to backfill all historical data, a sync should be initiated after your clear succeeds.
 
-## Performing a Reset
-To perform a full reset that resets all your streams, select `Reset your data` in the UI on a connection's status or job history tabs by selecting the three grey dots next to "Sync now". 
+## Clearing your data
```
Comment (tealjulia) [inline docs/operator-guides/reset.md:13]: nit: above, it mentions the ability to clear "some" streams... is it worth mentioning somewhere that this is only supported via the API?

File: docs/operator-guides/reset.md [hunk 2]
```diff
@@ -2,29 +2,29 @@
...
+To perform a full removal of the data for all your streams, select `Clear your data` in the UI on a connection's `Settings` tab and click "Clear data". Confirm the selection to remove all previously synced data from the destination for that connection.
```
Comment (tealjulia) [inline docs/operator-guides/reset.md:14]:
```suggestion
To perform a full removal of the data for all your streams, navigate to a connection's `Settings` tab and click "Clear data". Confirm the selection to remove all previously synced data from the destination for that connection.
```

Instructions have an extra click listed.

File: docs/operator-guides/reset.md [hunk 3]
```diff
@@ -2,29 +2,29 @@
...
 
-Similarly to a sync job, a reset can be completed as successful, failed, or cancelled. To resolve a failed reset, you should manually drop the tables in the destination so that Airbyte can continue syncing accurately into the destination. 
+Similarly to a sync, a clear can be completed as successful, failed, or cancelled. To resolve a failed clear, you should manually drop the tables in the destination so that Airbyte can continue syncing accurately into the destination. 
 
-## Reset behavior
-When a reset is successfully completed, all the records are deleted from your destination tables (and files, if using local JSON or local CSV as the destination), and then the next sync will begin.
+## Clear behavior
+When clearing data is successfully completed, all the records are deleted from your destination tables (and files, if using local JSON or local CSV as the destination), and then the next sync will begin.
```
Comment (tealjulia) [inline docs/operator-guides/reset.md:27]:
```suggestion
When clearing data is successfully completed, all the records are deleted from your destination tables (and files, if using local JSON or local CSV as the destination).
```

File: docs/operator-guides/reset.md (hunk 1 above)
Comment (nataliekwong) [reply docs/operator-guides/reset.md:13]: Added that note!

File: docs/operator-guides/assets/clear-example.png
Comment (tealjulia) [inline docs/operator-guides/assets/clear-example.png:1]: Note: I'm adjusting this wording in https://github.com/airbytehq/airbyte-platform-internal/pull/12152 after feedback from support

Comment (nataliekwong) [reply docs/operator-guides/assets/clear-example.png:1]: I removed the image for now!

**Review by teallarson — APPROVED** [review]

**Pull request merged**